flink-runtime/src/test/java/org/apache/flink/runtime/metrics/ReporterSetupTest.java

```diff
@@ -219,6 +222,21 @@ private static void assertReporter2Configured(ReporterSetup setup) {
 		Assert.assertEquals(TestReporter2.class.getName(), setup.getConfiguration().getString("class", null));
 	}
 
+	@Test
+	public void testVariableExclusionParsing() throws Exception {
+		final Configuration config = new Configuration();
+		config.setString(ConfigConstants.METRICS_REPORTER_PREFIX + "test." + ConfigConstants.METRICS_REPORTER_FACTORY_CLASS_SUFFIX, TestReporterFactory.class.getName());
+		config.setString(ConfigConstants.METRICS_REPORTER_PREFIX + "test." + ConfigConstants.METRICS_REPORTER_EXCLUDED_VARIABLES, "foo;bar");
+
+		final List<ReporterSetup> reporterSetups = ReporterSetup.fromConfiguration(config);
+
+		assertEquals(1, reporterSetups.size());
+
+		final ReporterSetup reporterSetup = reporterSetups.get(0);
+
+		assertThat(reporterSetup.getExcludedVariables(), hasItems(ScopeFormat.asVariable("foo"), ScopeFormat.asVariable("bar")));
```

Comment:
nit: One could introduce variables for `foo` and `bar` so that we don't have two places where they are defined.